app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Register and create an application as a developer
+---
+
+This guide explains how developers can get started with the developer platform by registering and creating an application. 
+
+The Dev Portal enables developers to integrate applications with APIs from third-party providers. For instance, if you are developing for a company that requires access to two distinct APIs from a third-party provider, you would need to set up an application within the Dev Portal that consumes both APIs.
+
+The following diagram shows how you can register your app in Dev Portal:
+
+{% mermaid %}
+flowchart TD
+    A[Sign up for Dev Portal] --> B(Create an application)
+    B --> C(Add products to the application)
+    C --> D(Create application credentials)
+    D --> |Generate in Dev Portal| E[Use application] 
+    D --> |Create in IdP| E
+{% endmermaid %}
+
+> **Figure 1:** To create an application against APIs in the Dev Portal, you must first register an account. You can use APIs in the developer platform to create your own app and register that app with the company's developer platform. Before you can use your app, you must create credentials to access the app.
+
+<details><summary>Key concepts</summary>
+
+{% capture konnect_concepts %}
+**Application:** An application enables developers to register and use APIs, whether they are your company’s own or those of a third-party provider. To facilitate diverse integrations, developers can create applications that consume multiple APIs from different providers, streamlining various services into a single offering.
+
+**Application credentials:** Credentials authorize developers to access an application. You can generate credentials automatically, or  manage them manually through an Identity Provider (IdP), like OIDC.
+
+**Developer platform:** The developer platform allows developers like you to locate, access, and consume APIs. Using the Dev Portal, you can browse and search API documentation, test API endpoints, and manage your own credentials. 
+{% endcapture %}
+
+{{ konnect_concepts | markdownify }}
+
+</details>
+
+## Register and create an application
+
+All developers must register through the {{site.konnect_short_name}} Dev Portal. A {{site.konnect_short_name}} admin can provide you with the correct registration URL.
+
+1. **Register or sign in to the developer platform**
+    
+    Navigate to Dev Portal and do one of the following:
+    * Register for access by navigating to the Dev Portal and creating an account.
+    * Sign in to the Dev Portal with single sign-on (SSO).
+
+    Once you're approved for access, you can create an application in Dev Portal.
```

Comment:
realizing we already have this https://docs.konghq.com/konnect/dev-portal/applications/dev-apps/ so now I'm a bit confused about this one pager. Can we end the doc here and direct people here: https://docs.konghq.com/konnect/dev-portal/applications/dev-apps/. Because there are 2 ways you can create an app in Portal